### PR TITLE
Add script to build packages from a specific tag

### PR DESCRIPTION
**Pull request by jrha**
This is an expanded form of the packaging section of `releaser.sh` which has been used to provide OS specific builds for the last couple of releases.

### CODE REVIEW

File: src/releasing/packager.sh [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  echo -e "\033[1;32mSUCCESS\033[0m  $1"
+}
+
+function echo_info {
+  echo -e "\033[1;34mINFO\033[0m  $1"
+}
+
+function exit_usage {
+    echo
+    echo "USAGE: releaser.sh RELEASE_NUMBER [RELEASE_CANDIDATE]"
```
Comment (stdweird) [inline src/releasing/packager.sh:24]: releaser -> packager? not sure about the rest of the help

File: src/releasing/packager.sh [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    exit 1
+fi
+
+PACKAGE_SUFFIX="el$VERSION_ID"
+
+echo_info "Set package suffix to $PACKAGE_SUFFIX"
+
+
+# Now define our environment
+REPOS_MVN="release aii CAF CCM cdp-listend configuration-modules-core configuration-modules-grid LC ncm-cdispd ncm-ncd ncm-query ncm-lib-blockdevices"
```
Comment (stdweird) [inline src/releasing/packager.sh:40]: normally, i'd ask to factor the common bits. not sure how easy it is to do?

Comment (jrha) [reply src/releasing/packager.sh:40]: I'll look at doing this in a different PR.

**Comment by jrha** [comment]
@stdweird are you happier with this one now?

**Review by stdweird — APPROVED** [review]

**Pull request merged**